test/index.ts

```diff
@@ -178,6 +179,22 @@ it('UniswapOracle on-chain hashes', async () => {
 	console.log({price})
 })
 
+it('timestamp', async () => {
+	const now = Math.round(new Date().getTime() / 1000)
+	const rpc = await createMemoryRpc(jsonRpcEndpoint, gasPrice)
+	const selfAddress = await rpc.addressProvider()
+
+	await fetch(`http://localhost:12340/${now}`)
+	await rpc.sendEth(selfAddress, 0n)
+	const firstBlock = await rpc.getBlockByNumber(false, 'latest')
+	expect(firstBlock!.timestamp).toEqual(new Date(now * 1000))
+
+	await fetch(`http://localhost:12340/${now + 1}`)
```

Comment:
Isn't this the likely behavior without faketime?

Reply:
Good point, I'll update it to match the mechanism we discussed, by taking the last block time and adding a bunch of seconds to it.

Reply:
Fixed.